ELISA Policies and Guidelines

```diff
@@ -0,0 +1,69 @@
+- https://github.com/elisa-tech/wg-medical-devices/commit/79290c454ab8f4f03560090843f84d13ed7b3360
+
+### Policy for tools and resources used for qualitative analysis
+
+All ELISA work products must be contributed and made available under GPL-2.0 and submitted with Developer Certificate of Origin (DCO) sign-off statements. This applies regardless of whether the work product is for tools or for user space code.
+
+If contributions are added to an existing tool, they are considered to be under the license associated with the existing project. 
+
+ELISA uses tools and references to develop example qualitative analysis for Automotive and Medical use-cases for the Linux kernel and provides resources for System integrators to use to analyze their systems.
+
+**Tools and references used in the qualitative analysis:**
+
+- Tools should satisfy the following criteria:
+	- Available now under GPL-2.0 or equivalent license
+	- Sources are publicly available and easily downloadable
+	- Actively maintained: Developers and maintainers are actively engaged in reviewing patches and pull requests for fixes and new features.
+	- When a tool is no longer maintained, it will be moved to an “attic” until a maintainer is identified.
+- WG leads are responsible for ensuring that any tool they use has been evaluated to confirm that it meets the above criteria.
+- Exceptions may be granted after reviewing the tool in the TSC if there are no other tools available and the development happens in open.
+- Evaluated tools must be documented in a publically available registry maintained by ELISA (e.g. a GitHub repo), which identifies the version(s) of the tool evaluated and records the results of the evaluation (e.g. was it selected, why it was rejected, analysis of tool against criteria from a safety standard, etc). 
+
+### Review Policies
+ELISA shall make tools and resources published via ELISA GitHub repositories.
+
+- Repositories must have a clearly identified main branch, which contains peer-reviewed and approved versions of the tools or resources
+- Changes to the main must be proposed in ‘working’ branches and reviewed by ELISA contributors via the GitHub Pull request (PR) process; direct changes to main will be disabled
+- Each repository will have one or more maintainers with the authority to approve Pull Requests (PRs) for inclusion in the main branch
+- Each repository must have a documented review policy and acceptance criteria for PRs, to provide guidance to contributors, maintainers and reviewers.
+- All new inbound code contributions must also be accompanied by a Developer Certificate of Origin (http://developercertificate.org) sign-off in the source code system that is submitted through a TSC-approved contribution process which will bind the authorized contributor and, if not self-employed, their employer to the applicable license.
+
+### Investigation on safety standards use in reference process
```

Comment:
Is this section the proper place for the document. We also kind of completed the reference process work, IIRC. Keeping it in is also not harming the content, so it may be an item for further discussion in a later stage.